### DISCUSSION TITLE
GraphQL Subscriptions

### DISCUSSION THREAD

**Discussion by VinayaSathyanarayana** [keystonejs/keystone#5312] [Feature requests]
Opening up the discussion thread to support GraphQL Subscriptions.

This will be a great feature to have.  Please provide your inputs /reactions / votes

**Comment by bladey** [comment]
Thanks for the input @VinayaSathyanarayana, we know GraphQL Subscriptions are a highly requested feature, have a look at this thread on Slack for more details - https://keystonejs.slack.com/archives/C01STDMEW3S/p1629254819176200?thread_ts=1629006123.011500&cid=C01STDMEW3S
